File: hh-back/src/main/java/gloit/hiperionida/helios/model/BancoModel.java
package gloit.hiperionida.helios.model;

import gloit.hiperionida.helios.util.model.AbsAuditoriaModel;

import lombok.*;
import jakarta.persistence.*;
import lombok.experimental.SuperBuilder;

import java.time.LocalDateTime;

@EqualsAndHashCode(callSuper = false)
@Entity
@Getter
@Setter
@SuperBuilder
@Table(name = "banco")
public class BancoModel extends AbsAuditoriaModel {
    private String banco;
    @Column(columnDefinition = "TEXT")
    private String notas;

    public BancoModel() {
    }

    public BancoModel(Long id, Long creadorId, LocalDateTime creada, Long modificadorId, LocalDateTime modificada, Long eliminadorId, LocalDateTime eliminada, String banco, String notas) {
        super(id, creadorId, creada, modificadorId, modificada, eliminadorId, eliminada);
        this.banco = banco;
        this.notas = notas;
    }
}

File: hh-front/src/models/banco_model.js
import { UsuarioModel } from './usuario_model'

export class BancoModel {
  constructor () {
    this.id = ''
    this.banco = ''
    this.notas = ''

    this.creador = new UsuarioModel()
    this.creada = ''
    this.modificador = new UsuarioModel()
    this.modificada = ''
    this.eliminador = new UsuarioModel()
    this.elimianda = ''
  }
}
